dev_scripts/qa.py

```diff
@@ -55,7 +55,20 @@
   - [ ] Run the Dangerzone tests.
   - [ ] Create an .rpm package and install it system-wide.
   - [ ] Test some QA scenarios (see [Scenarios](#Scenarios) below).
-
+- [ ] Create a test build in the most recent Fedora template in Qubes OS (Fedora 38 as of
+  writing this) and make sure it works:
+  - [ ] Create a new development environment with Poetry.
+  - [ ] Build the container image and ensure the development environment uses
+        the new image.
+  - [ ] Run the Dangerzone tests and confirm that the tests are ran using diposable qubes
+        instead of containers
```

Comment:
You probably mean containers instead of disposable qubes, right?

Reply:
Thanks, but I see a typo ("containers qubes").

Reply:
should be resolved in https://github.com/freedomofpress/dangerzone/pull/464/commits/c3e439d4c1f9ff91dbb449cc1d456bb8b5087574